dora/core/server/worker/src/test/java/alluxio/worker/dora/PagedDoraWorkerTest.java

```diff
@@ -1039,15 +1038,11 @@ public void searchPagesByPrefix() throws Exception {
   }
 
   private void checkPrefixSearch(UfsUrl rootUrl, UfsUrl prefixUrl) {
```

Comment:
You should remove the `@Ignore` annotation of `searchPagesByPrefix()`, so the test will be run on github CICD.